Spore ModAPI/Spore/App/Property.h

```diff
@@ -273,6 +273,12 @@ namespace App
 		/// If the property is not of the specified type, a pointer to a default value will be returned.
 		///
 		void* GetValue();
+
+		///
+		/// Returns the number of items inside the array of the property it read.
+		/// If the property is not an array, I have no clue what will happen.
+		///
+		size_t GetItemCount();
```

Comment:
Change the documentation to this:
```cpp
///
/// Only for array properties, returns the number of items inside the array.
/// If the property is not an array, it might return any number.
///
```